python/eos/data/native.py

```diff
@@ -340,7 +340,7 @@ def create(path, parameters, proposal, sigma_test_stat=None, samples=None, weigh
 
         # The test statistics defaults to two empty lists
         description['test statistics'] = { "sigma": [], "densities": [] }
-        if sigma_test_stat and samples and weights:
+        if not sigma_test_stat is None and not samples is None and not weights is None:
```

Comment:
That's not easy to read. Does the following work?
```python
if sigma_test_stat is not None and samples is not None and weights is not None:
```

Reply:
It does :)